script.go

```diff
@@ -76,8 +77,12 @@ func File(path string) *Pipe {
 }
 
 // FindFiles creates a pipe listing all the files in the directory dir and its
-// subdirectories recursively, one per line, like Unix find(1). If dir doesn't
-// exist or can't be read, the pipe's error status will be set.
+// subdirectories recursively, one per line, like Unix find(1).
+// If an error occurs while reading the directory, the behaviour is as follows:
```

Comment:
This is good, but I think we can be a bit more concise. What about:

```
// Errors are ignored unless no files are found (in which case the pipe's error
// status will be set to the last error encountered). 
```

Reply:
@bitfield done!